### ISSUE TITLE
Not working

### ISSUE THREAD

**Issue by lpender** [typicode/husky#305]
Hi, thanks for this!

I wish I could provide a better error that "not working" but the docs make it out to be so simple that I'm not sure what to even argue with...

`package.json`

```
  "husky": {
    "hooks": {
      "pre-commit": "npm test",
      "precommit": "npm test"
    }
  },
  "scripts": {
    "pre-commit": "npm test",
    "precommit": "npm test",
  },
  "devDependencies": {
    "husky": "^1.0.0-rc.9",
  }
```

I'm not seeing any message, no error, nothing. It's just not running.

Is there a way of troubleshooting this?

**Comment by myu20** [comment]
I'm having the same issue. If I run `./.git/hooks/pre-commit` from my package root everything works fine, but nothing runs when I commit.

**Comment by typicode** [comment]
Hi @lpender, @myu20,

Thanks for raising this issue. Don't really know why it's not working. What's your OS? Are you using a particular Git client/GUI? Does `pre-commit` file have `0755` permissions (can read and execute)?

If you edit `.git/hooks/pre-commit` and add some random message:

```
#!/bin/sh
# husky
# v1.0.0-rc.9 darwin

echo "hello world"

...
```

Do you see the `hello world` message you've added when committing?

**Comment by lpender** [comment]
Hi, I think I may have figured it out.

I had made changes to `pre-commit` under `.git/hooks/`.

It looks like `husky` works by adding/overwriting these files (which I didn't realize/might be good to add to the README), and it was unable or unwilling to overwrite the file since I had made changes to it.

Removing this file and reinstalling or reinitializing should fix it--I'm not sure. I've chosen to go another route for now.

Thanks!
